applications/GeoMechanicsApplication/custom_elements/small_strain_U_Pw_diff_order_element.cpp

```diff
@@ -584,43 +536,32 @@ void SmallStrainUPwDiffOrderElement::CalculateOnIntegrationPoints(const Variable
                        std::multiplies<>{});
 
         // Loop over integration points
+        const SizeType dimension = r_geometry.WorkingSpaceDimension();
         for (unsigned int g_point = 0; g_point < mConstitutiveLawVector.size(); ++g_point) {
             // compute element kinematics (Np, gradNpT, |J|, B, strains)
             this->CalculateKinematics(Variables, g_point);
             Variables.B = b_matrices[g_point];
 
-            // Compute FluidFlux vector q [m/s]
-            const SizeType Dim       = r_geometry.WorkingSpaceDimension();
-            const SizeType NumUNodes = r_geometry.PointsNumber();
-
-            Vector   BodyAcceleration = ZeroVector(Dim);
-            SizeType Index            = 0;
-            for (SizeType i = 0; i < NumUNodes; ++i) {
-                for (unsigned int idim = 0; idim < Dim; ++idim)
-                    BodyAcceleration[idim] += Variables.Nu[i] * Variables.BodyAcceleration[Index++];
+            Vector   body_acceleration = ZeroVector(dimension);
+            SizeType Index             = 0;
+            for (SizeType i = 0; i < r_geometry.PointsNumber(); ++i) {
+                for (unsigned int idim = 0; idim < dimension; ++idim) {
+                    body_acceleration[idim] += Variables.Nu[i] * Variables.BodyAcceleration[Index];
+                    Index++;
```

Comment:
In general, pre-increment operators should be preferred over post-increment operators (for performance reasons):
```suggestion
                    ++Index;
```

Reply:
done